sdk/storage/.editorconfig

```diff
@@ -0,0 +1,165 @@
+###############################

```

Comment:
This should eventually match https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/.editorconfig where we are trying to enforce a common set of standards.

Reply:
Are we inheriting a config in a higher directory?  I'm guessing not-really, given the fact our sln is rooted lower.  Perhaps an explicit inclusion?

Reply:
Figuring out how and when to align on these is a thing we should chat about at our next sync.  It might be enough work that it ends up as an item on the roadmap.